src/main/java/gui/viewmodel/MonthlyCalendarViewModel.java

```diff
@@ -3,26 +3,37 @@
 import com.calendarfx.model.Entry;
 import datagateway.Observer;
 import datagateway.event.EventReader;
```

Comment:
Using `EvntInfo`  instead?

Reply:
both will be needed as we are receiving an EventReader from the observable repo when getting updates.